app/src/main/java/org/dasfoo/rover/android/client/util/L.java

```diff
@@ -0,0 +1,202 @@
+package org.dasfoo.rover.android.client.util;
+
+import android.util.Log;
+
+/**
+ * Log wrapper. To disable logging L.isLogEnable = false; To set min log level,
+ * when log enabled set L.logLevel = Log.VERBOSE;
+ */
+public class L {
+    /**
+     * It enables/disables logs.
+     */
+    public static boolean isLogEnable = false;
+
+    public static int logLevel = Log.VERBOSE;
```

Comment:
Android [`Log`](https://developer.android.com/reference/android/util/Log.html) class has a `SUPPRESS` level, would it make sense to use it instead of `isLogEnable`?